errors.go

```diff
@@ -1,4 +1,4 @@
-package storage
+package elrond_go_storage
```

Comment:
perhaps move this file into `common/errors.go` or rename the package

Reply:
moved to a new package